src/js/shield_canvas_draw.js

```diff
@@ -711,5 +656,12 @@ export function registerDrawFunction(name, fxn) {
 }
 
 //Built-in draw functions (standard shapes)
-registerDrawFunction("roundedRectangle", roundedRectangle);
+registerDrawFunction("diamond", diamond);
 registerDrawFunction("ellipse", ellipse);
+registerDrawFunction("escutcheon", escutcheon);
+registerDrawFunction("hexagonVertical", hexagonVertical);
+registerDrawFunction("hexagonHorizontal", hexagonHorizontal);
+registerDrawFunction("octagonVertical", hexagonVertical);
```

Comment:
Shouldn't both of these be `octagonVertical`?

Reply:
Yup!  Fixed.